p2p/rlpx/rlpx_test.go

```diff
@@ -383,12 +382,8 @@ func BenchmarkHandshakeRead(b *testing.B) {
 }
 
 func BenchmarkThroughput(b *testing.B) {
-	pipe1, pipe2, err := pipes.TCPPipe()
```

Comment:
It's useful to have TCPPipe because it buffers the input to some extent, unlike `net.Pipe` which is implemented purely in Go and does not buffer.

Reply:
Okay I see. I added back the `TCPPipe` implementation.